packages/status-react/src/routes/chat/index.tsx

```diff
@@ -57,10 +81,38 @@ const Body = () => {
     <>
       <ContentWrapper ref={contentRef}>
         <ChatStart chatId={chatId} />
-        {messages.data.map(message => (
-          <ChatMessage key={message.messageId} message={message} />
-        ))}
+        {/* temporary */}
+        <div style={{ textAlign: 'center' }}>
+          <button
+            type="button"
+            onClick={messages.fetchMore}
+            disabled={messages.loading}
+          >
+            {messages.loading ? 'Loading...' : 'Load more'}
+          </button>
+        </div>
       </ContentWrapper>
+      {messages.data.map((message, index) => {
+        const sentDate = new Date(Number(message.clock))
```

Comment:
Non-blocking: why not `timestamp` since this is not for ordering 🤔?

Reply:
Agreed.